### ISSUE TITLE
require failure with 0.0.42

### ISSUE THREAD

**Issue by davidwood** [bigeasy/timezone#250]
Wit the 0.0.42 of timezone, an error is thrown when the module is required:

```javascript
module.js:340
   throw err;
   ^
Error: Cannot find module 'timezone/loaded'
   at Function.Module._resolveFilename (module.js:338:15)
   at Function.Module._load (module.js:280:25)
   at Module.require (module.js:364:17)
   at require (module.js:380:17)
```
It looks as if the [0.0.42 tarball](http://registry.npmjs.org/timezone/-/timezone-0.0.42.tgz) contains the source of the module, but not the build.

**Comment by bigeasy** [comment]
Thank you, will fix.

**Comment by bwinter** [comment]
++ having the same issue

**Comment by bigeasy** [comment]
I've published 0.0.43 which should be correct. Thank you.
